draft-ietf-cose-tsa-tst-header-parameter.md

```diff
@@ -199,6 +199,39 @@ IANA is requested to add the COSE header parameters defined in {{tbl-new-hdrs}}
 
 --- back
 
+# Examples
+
+## CTT
+
+> NOTE: The following uses the currently unassigned 50 as TBD2.  This example must be regenerated once IANA has allocated the codepoint.
```

Comment:
```suggestion
> The following examples use the currently unassigned 54424432 as TBD2.  This example must be regenerated once IANA has allocated the codepoint.
```

Reply:
if this is okay, it could be propagated throughout the example

Reply:
:-( sad face emoji

Reply:
if IANA agrees on 50/51 for TBD2/TBD1 we might get away without having to recompute the examples...